app/models/claim_student_loan_details_updater.rb

```diff
@@ -32,11 +36,18 @@ def eligibility_student_loan_attributes
 
   def claim_student_loan_attributes
     {
-      has_student_loan: student_loans_data.found_data?,
+      has_student_loan: student_loans_data.has_student_loan?,
       student_loan_plan: student_loans_data.student_loan_plan
     }
   end
 
+  def claim_student_loans_policy_student_loan_attributes
```

Comment:
I didn't want to change these values for the TSLR journey - as it impacts how that journey works.